probe-rs/src/config/registry.rs

```diff
@@ -123,17 +123,10 @@ struct Registry {
 
 impl Registry {
     fn from_builtin_families() -> Self {
-        #[cfg(feature = "builtin-targets")]
```

Comment:
I think this feature might be keep-worthy :)

Reply:
The feature still exists, it's just completely inside the build script